README.md

```diff
@@ -82,21 +82,25 @@ After you obtain a Fluo YARN distribution tarball, follow these steps to install
 
 ## Start Fluo application in YARN
 
-Follow the instructions below to start a Fluo application which contain an oracle and multiple workers.
+Before you can launch a Fluo application in YARN, you should follow Fluo's [install] and [application] instructions
```

Comment:
It shouldn't be repeated but I want to make sure users new initialize their application first.  See updated text in 7f8a24434d8cab